02_configure_host.sh

```diff
@@ -18,6 +18,7 @@ fi
 ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -e "working_dir=$WORKING_DIR" \
     -e "num_nodes=$NUM_NODES" \
+    -e "num_redfishvmnodes=$NUM_REDFISHVMNODES" \
```

Comment:
adding two extra nodes does not seem efficient cannot we have one node and another refishvmnode for integration tests for example ?

Reply:
changed it kindly check.